### PR TITLE
🌱 Run prow triggerable basic e2e on project-infra and fix branch issue

### PR DESCRIPTION
**Pull request by mboukhalfa**
this PR fix the branch issue when triggering basic e2e from prow and make it also runnable on project infra

### CODE REVIEW

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

File: prow/manifests/overlays/metal3/config.yaml [hunk 1]
```diff
@@ -1149,6 +1149,16 @@ presubmits:
       always_run: false
       # Until we have checked that it works, keep it optional
       optional: true
+    # name: {job_prefix}-{proj}-{capm3_target_branch}-e2e-basic-test-{image_os}
+    - name: metal3-project_infra-main-e2e-basic-test-centos
+      branches:
+        - main
+        - release-1.6
```
Comment (kashifest) [inline prow/manifests/overlays/metal3/config.yaml:1156]: release-1.5?

**Review by kashifest — COMMENTED** [review]
/approve

File: prow/manifests/overlays/metal3/config.yaml (hunk 1 above)
Comment (kashifest) [reply prow/manifests/overlays/metal3/config.yaml:1156]: as talked offline we are still unsure and we need to test the branches listed here first.

**Pull request merged**